apps/frontend/src/app/poll/choose-events/choose-events.component.ts

```diff
@@ -202,13 +206,23 @@ export class ChooseEventsComponent implements OnInit {
     } else if (maxParticipants && this.participants.length >= maxParticipants) {
       this.closedReason = 'This poll has reached it\'s maximum number of participants. You can no longer submit your vote.';
       this.showResults = true;
+    } else if (this.poll?.settings?.showResult === ShowResultOptions.NEVER as ShowResultOptions) {
+      this.closedReason = undefined;
+      this.showNever = !this.userVoted() && !this.isAdmin;
+      if (!this.isAdmin) {
+        this.participants = this.participants.filter(p => p.token === this.token);
+      }
```

Comment:
This needs to be done by the backend!

Reply:
You are absolutly right! My bad. Fixed in be56f71.